manifests/charts/istio-cni/templates/network-attachment-definition.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: istio-cni
+  namespace: default
```

Comment:
From the doc:

"to be present in the application namespace in order to invoke the istio-cni plugin"

Is this not accurate with the `ns/name` format, and allows it to be anywhere?

Regardless, putting it in `default` seems a bit odd. I would expect it to be thesame as where CNI is deployed?